packages/forma-36-react-components/src/components/ControlledInput/__snapshots__/ControlledInput.test.tsx.snap

```diff
@@ -17,11 +19,13 @@ exports[`renders the component with a value 1`] = `
 exports[`renders the component with all required props 1`] = `
 <input
   aria-label="ControlledInput"
-  className="ControlledInput"
+  className=""
```

Comment:
Changes in this file are caused by the error above.

Reply:
oh dammit!

Reply:
ok fixed thanks for the heads up!